tests/slurm_plugin/test_common.py

```diff
@@ -0,0 +1,610 @@
+import os
+from unittest.mock import MagicMock, call
+
+import pytest
+
+from assertpy import assert_that
+from common.schedulers.slurm_commands import SlurmNode
+from slurm_plugin.common import InstanceLauncher, _get_instance_ids_to_nodename, delete_instances
+from tests.common import MockedBoto3Request
+
+
+@pytest.fixture()
+def boto3_stubber_path():
+    # we need to set the region in the environment because the Boto3ClientFactory requires it.
```

Comment:
I think this does not apply in the node package. I think it comes from pcluster batch CLI tests

Reply:
This fixture is used by the boto3 stubber fixture to specify the path of the boto3 import to mock, so I think it is required.

Reply:
the fixture is required. I was referring to the comment and the line below related to Boto3ClientFactory

Reply:
The region is actually required, or else we will get an error like this: https://travis-ci.org/github/aws/aws-parallelcluster-node/jobs/712808915